### ISSUE TITLE
Allow escorts to attack one another.

### ISSUE THREAD

**Issue by mdsmestad** [endless-sky/endless-sky#748]
It seems to me that I should be able to tell my escorts to attack one another if I tell them to. This is also have other uses, if healing weapons were added to the game, for instance. Expanding on this concept, I'd love for the AI to know to attack friendly targets when there weapon output has negative hull or shield damage (healing beams & such)

**Comment by LocalGod79** [comment]
you used to be able to tell your escorts to focus their fire on one of your own escorts - has this changed?

**Comment by Bladewood** [comment]
Healing beams. Brilliant.

**Comment by LocalGod79** [comment]
maybe also a personality type request "mutinous", for healing ships?

**Comment by thenerdfreak** [comment]
I already made a plugin for a healing beam somewhere on github, prolly got sent to forums tho . and yeah this used to be a thing and I miss it a lot

**Comment by lehgogh** [comment]
(I didn't read that @thenerdfreak had already made something similar to this until I already made it, but that's okay.)

I made [a healing blaster plugin](https://github.com/lehgogh/endless-sky-plugins/tree/master/ES-Nanobot-Weapons), if anyone's interested in trying it. What I'm concerned about, however, is that escorts will attack enemies with it (which would heal them) but not allies (which defeats the point of such a weapon). The only way to use it effectively is by:
- only giving one to the flagship
- adding mechanics which make escorts only target allies with healing weapons

Also, this would make hailing for repairs obsolete in the end-game, which isn't really bad.
